Web/Pages/Transactions/TransactionForm.razor

```diff
@@ -293,6 +293,7 @@
     }
 
     MudDatePicker _picker = new();
+    private int maxFileSize = 1024 * 1025 * 10;
```

Comment:
why 1025?

Reply:
Because I 1-upped it.
Nah, should be 1024*1024*10